tracer/build/_build/Build.Steps.cs

```diff
@@ -567,6 +568,32 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                     .SetOutput(MonitoringHomeDirectory / framework)));
         });
 
+    Target PublishManagedTracerForAwsLambda => _ => _
+        .Unlisted()
+        .Requires(() => PublishReadyToRun == false || RuntimeIdentifier != null) // RuntimeIdentifier is required if PublishReadyToRun is enabled
+        .After(Clean, Restore)
+        .Executes(() =>
+        {
+            var framework = TargetFramework.NET6_0;     // always target .NET 6 on AWS Lambda
+            var readyToRun = PublishReadyToRun ?? true; // default to ReadyToRun on AWS Lambda unless explicitly disabled
```

Comment:
It would be better to default this on the `PublishReadyToRun` parameter IMO.

Reply:
done!